doc/manuscript_Rmd/manuscript.Rmd

```diff
@@ -7,9 +7,16 @@ author:
 address:
   - code: a
     address: "University of California, Berkeley"
+author:
```

Comment:
Actually, this doesn't work. You would need to add your name l.6, just under my email address.

Reply:
Ah, thanks for the clarification! I think I've fixed the problem.